elixir/apps/domain/lib/domain/application.ex

```diff
@@ -61,7 +61,10 @@ defmodule Domain.Application do
     %{
       id: Domain.Events.ReplicationConnection,
       start: {Domain.Events.ReplicationConnection, :start_link, [init_state]},
-      restart: :transient
+      restart: :transient,
```

Comment:
Is this some kind of exponential backoff? Or how is the restart spaced out?

Reply:
The Supervisor module is basic - evenly spaced.

But this is what we want during a deploy.

Elixir has much more advanced supervisors to manage the process tree if we need to use them.